source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -278,8 +293,44 @@ HTTP Webhook Audit Log Target
 
       Requires specifying :mc-conf:`~audit_webhook.client_cert`.
 
-      This setting corresponds to the
-      :envvar:`MINIO_AUDIT_WEBHOOK_CLIENT_KEY` environment variable.
+      This setting corresponds to the :envvar:`MINIO_AUDIT_WEBHOOK_CLIENT_KEY` environment variable.
+
+   .. mc-conf:: queue_dir
```

Comment:
I don't see a `MINIO_AUDIT_WEBHOOK_QUEUE_SIZE`, I'm confused that you can set a path but not the size of the queue to write there.

Reply:
https://github.com/minio/minio/blob/master/internal/logger/config.go#L73-L79

Looks like it's in the code, but we missed it in the actual docs - which...happens. You can add it if you want in this run.